modin/pandas/test/test_io.py

```diff
@@ -1132,6 +1132,30 @@ def test_read_csv_empty_frame(self):
             index_col="col1",
         )
 
+    @pytest.mark.parametrize(
+        "skiprows",
+        [
+            lambda x: x > 20,
+            lambda x: True,
+            lambda x: x in [10, 20],
+            pytest.param(
+                lambda x: x << 10,
+                marks=pytest.mark.skipif(
+                    condition="config.getoption('--simulate-cloud').lower() != 'off'",
+                    reason="The reason of tests fail in `cloud` mode is unknown for now - issue #2340",
```

Comment:
did you test that it really fails in the cloud mode?

Reply:
Yes, test in this case fails with error
`RecursionError: maximum recursion depth exceeded while calling a Python object`